keyboards/handwired/108keyTrackpoint/keymaps/default/keymap.c

```diff
@@ -0,0 +1,14 @@
+#include "108keyTrackpoint.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = KEYMAP (
```

Comment:
The layout macros references throughout need to be updated as well.

```suggestion
  [0] = LAYOUT(
```